Appraisals

```diff
@@ -95,7 +95,7 @@ RAILS_VERSIONS = %w(
   4.2.5.1
   4.2.5.2
   4.2.6
-  5.0.0.beta3
+  5.0.0.1
```

Comment:
Add both. 

> why you want to test this library against Rails version that should never be used?

To be sure the behavior is the same accross all versions of Rails.